lib/utils/scheduler.js

```diff
@@ -1,16 +1,18 @@
 const moment = require('moment');
 
-module.exports = (interval = 'day') => {
-
+module.exports = (interval = 'minute') => {
   let lastCall;
   const tasks = [];
+  const aMinute = 60 * 1000; // also minimum interval
 
   setInterval(() => {
-    if (!lastCall || lastCall.isBefore(moment(), interval)) {
+    const now = moment();
+    // will run once a minute after starting and then again each interval
```

Comment:
I think this is misleading - it will _check_ whether it should run once a minute, but only executes once an hour - I'd either reword or remove the comment

Reply:
it will always run at least once on startup after 60 seconds pass because lastCall will be `undefined` - the wording isn't great though.